compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -441,6 +441,8 @@ public void visit(BLangObjectTypeNode objectTypeNode) {
         // Add object level variables to the init function.
         Map<BSymbol, BLangStatement> initFunctionStmts = objectTypeNode.initFunction.initFunctionStmts;
         objectTypeNode.fields.stream()
+                // skip if the field is already have an value set by the constructor.
+                .filter(field -> !initFunctionStmts.containsKey(field.symbol))
```

Comment:
Shall we indent this line so that it comes under '.stream' and do the same for the subsequent lines? 
```ballerina
objectTypeNode.fields.stream()
                     .filter(field -> !initFunctionStmts.containsKey(field.symbol))

Reply:
2 indentations (8 spaces) is the wso2 default for wrapped lines...